tests/core/command/command.js

```diff
@@ -302,5 +302,106 @@ bender.test( {
 		editor._.elementsPath.onClick( 0 );
 
 		wait();
+	},
+
+	// #456
+	'test basic behaviour of getLabel': function() {
+		bender.editorBot.create( {
+			name: 'getLabel1',
+			config: {
+				on: {
+					// This allows on setting up nice enviornment before asserts.
+					'pluginsLoaded': function( evt ) {
+						var editor = evt.editor;
+
+						editor.addCommand( 'custom1', {
+							exec: function() {
+								return;
```

Comment:
Unnecessarily expressive function. Use just `function() {}`. This is repeated few times in this file.